docs/toc.yml

```diff
@@ -103,8 +104,9 @@
         href: design/office-ui-fabric.md
   - name: Develop
```

Comment:
Should **Develop** come earlier in the TOC. I can imagine the ordering being:
\> Office Add-ins guidance
     About Office Add-ins
  \> Overview
  \> Develop
  \> Concepts
  \> User Interface
...

Reply:
Per our discussion, we'll potentially revisit this TOC sequence later on, as we refactor contents of the various sections.